generate_training_data_scripts/README.rst

```diff
@@ -0,0 +1,75 @@
+
+This produces two files: *inspire_core_records.json* and *inspire_noncore_records.json*. We will later combine this data with the Rejected records data.
+
+Here, we consider only data from 2016 onwards since before that the curation rules for classification as Core, Non-Core, and Rejected were different. However, the user is free to modify the earliest_date field in the script to specify a starting data of their choice.
+
+Generate Rejected data
+^^^^^^^^^^^^^^^^^^^^^^
+
+The data for Rejected articles is extracted from in a hackish way. The workflows themselves need to be modified in our local inspire-next setup. First, the file *inspire-next/inspirehep/modules/workflows/workflows/article.py* needs to be modified as specified in ``article.py``. We need to add another file *inspire-next/inspirehep/modules/workflows/tasks/makejson.py* with the contents of ``makejson.py``.
+
+First, we need to deploy the harvest spiders. This can be done from the *inspire-next* instance folder:
+
+::
+
+    docker-compose -f docker-compose.deps.yml run --rm scrapyd-deploy
+
+To trigger the harvest, we run the following from the inspirehep cli:
+
+::
+
+    inspirehep crawler schedule arXiv article --kwarg 'from_date=2018-11-06' --kwarg 'until_date=2018-11-07' --kwarg 'sets=cs,econ,eess,math,physics,physics:astro-ph,physics:cond-mat,physics:gr-qc,physics:hep-ex,physics:hep-lat,physics:hep-ph,physics:hep-th,physics:math-ph,physics:nlin,physics:nucl-ex,physics:nucl-th,physics:physics,physics:quant-ph,q-bio,q-fin,stat'
+
+Here, we need to specify the *from_date* and *until_date*. **Important**: Make sure that the dates (especially the until_date) correspond to the date of the INSPIRE dump used for record migration. This is important as later on, we also need to generate the list of Core and Non-Core records from the same INSPIRE dump and any records harvested from after that date will be considered Rejected and that will corrupt our dataset.
+
+Since the harvest will generate a large number of files, it's reasonable to delete files associated with records from the workflows which have already finished. We can use any job or task scheduler for this as we need to delete the content created in *$DOCKER_DATA/tmp/virutalenv/var/data/workflows/files"* periodically, where *$DOCKER_DATA* is the environment variable corresponding to our *inspire-next* docker data. We can use *crontab* as the task scheduler in linux. In the bash terminal:
+
+::
+
+    crontab -e
+
+This will open our favorite text editor (or we'll be required to set it). Add the following line at the end of the file and save and quit:
+
+::
+
+    */15 * * * * find $DOCKER_DATA/tmp/virtualenv/var/data/workflows/files/* -mmin +30 -delete
+
+This will schedule a task to run every 15 minutes which will find and delete all files created before the last 30 minutes. It's recommended to schedule the cronjob after starting the harvests since the first harvests and workflows can take a few minutes to start.
```

Comment:
Maybe we can explain exactly what each parameter of the command means, so the user can modify it according to his computer? Maybe he has close to no free space and they want to delete it more often, or files that are less old.
They can also read it up themselves, as there is a lot of information online about this and it's not exactly our job to teach about cronjobs, but maybe the user will be happier if we get as specific as we can. Either way, I leave the choice to you, if you want to add that information or not.

Reply:
I'll try to be a bit more elaborate about this.